src/vec_buffers.h

```diff
@@ -11,34 +11,38 @@
 namespace nttec {
 namespace vec {
 
-/*
- * Wrapper of a vector of `n` buffers each of `size` elements of type T
- *  std::vector<T*> vec(n)
- *    vec[i] = T[size];
+/** A vector of `n` buffers (array of T).
+ *
+ * A Buffers contains pointers to `n` buffers such as
+ *
+ * Example:
+ *
+ * We have a set of `N` independent buffers (+ a buffer of zeros):
```

Comment:
`(+ a buffer of zeros)` should not be there, shouldn't it?

Reply:
Yup, I'll fix this.